cwltool/main.py

```diff
@@ -662,10 +662,10 @@ def loc_to_path(obj):
             prov_dep = prov_dependencies[1]
             assert prov_dep
             runtimeContext.research_obj.generate_snapshot(prov_dep)
-            #for input file dependencies
-            if input_for_prov:
-                runtimeContext.research_obj.generate_snapshot(input_for_prov)
             #NOTE: keep these commented out lines to evaluate tests later
+            #for input file dependencies
+            #if input_for_prov:
+                #runtimeContext.research_obj.generate_snapshot(input_for_prov)
             #if job_order_object:
```

Comment:
Can these commented out lines  be removed altogether?

Reply:
Done.